polkadot/cli/src/lib.rs

```diff
@@ -95,6 +95,22 @@ impl substrate_rpc::author::AuthorApi for RpcTransactionPool {
 	}
 }
 
+struct Configuration(service::Configuration);
+
+impl substrate_rpc::system::SystemApi for Configuration {
+	fn system_version(&self) -> substrate_rpc::system::error::Result<String> {
```

Comment:
It would be nice to have a `system_name` as well. (i.e. the JS client version don't match up with these, however the version + name is a proper identifier)